index.d.ts

```diff
@@ -262,6 +262,11 @@ declare namespace execa {
 		The signal that was used to terminate the process.
 		*/
 		signal?: string;
+
+		/**
+		Original error message when the child process exits due to an error or a timeout.
+		*/
+		originalMessage?: string;
```

Comment:
I don't think it should be placed here as it's only available on errors, not also the normal return value.

Reply:
Fixed in b0ba95aa885ce7531a5b21c09a2c47686dec98b0